travis/test_server.py

```diff
@@ -302,7 +298,13 @@ def main(argv=None):
             test_loghandler = 'openerp.tools.yaml_import:DEBUG'
     odoo_full = os.environ.get("ODOO_REPO", "odoo/odoo")
     server_path = get_server_path(odoo_full, odoo_version, travis_home)
-    script_name = get_server_script(odoo_version)
+    # Normalize on v10 odoo-bin command
+    try:
+        shutil.copy(
+            os.path.join(server_path, 'openerp-server'),
+            os.path.join(server_path, 'odoo-bin'))
```

Comment:
@dreispt I understand your concerns. However, in a certain way at least for the local machine case, it is most probably necessary to construct a dedicated testbed, which is initiated for example by a rsync command. It could be a problem if you bind mount read only indeed. But, if we are operating on read only filesystems, the whole `travis_install_nightly` will not work...
I think the solution is a "testbed", a word which i already used several times now:
A dedicated folder with predefined permissions and for single use. So stateful changes, do not matter. BTW, there are other stateful changes relevant in testing: Think you want to inhabilitate a testing of a core module in order to replace it with a custom test via `sed` command. So I guess the concept of (one way) testbed is pretty appropriate and stateful changes are not of concern.